src/test/java/nearsoft/academy/bigdata/recommendation/ManageList.java

```diff
@@ -0,0 +1,50 @@
+package nearsoft.academy.bigdata.recommendation;
+
+import java.util.HashMap;
+
+/**
+ * Created by AMDA on 30/03/2017.
+ */
+class ManageList
+{
+    public HashMap<String, Long> productList, userList;
```

Comment:
It's usually better to use interfaces unless you need an specific implementation